m4_sessions/1808-inning/surprise_of_happiness.md

```diff
@@ -1,18 +1,30 @@
 ## Group Members:
+Haley Mesander
+Sabrina Robinson
+Tory Dannenberg
 
-
-
-## Name of Session: 
+## Name of Session: Surprise of Happiness
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+     What makes you happy? That's what you think...
```

Comment:
I might add just a bit more to this...
"That's what you think... Come to this session to understand how you can increase happiness at Turing."